aspnetcore/razor-components/index.md

```diff
@@ -80,44 +52,85 @@ When this component is used elsewhere in the app, IntelliSense speeds developmen
+
+![Blazor runs .NET code in the browser with WebAssembly.](index/_static/blazor.png)
+
+When a Blazor app is built and run in a browser:
+
+* C# code files and Razor files are compiled into .NET assemblies.
+* The assemblies and the .NET runtime are downloaded to the browser.
+* Blazor uses JavaScript to bootstrap the .NET runtime and configures the runtime to load assemblies. Document Object Model (DOM) manipulation and browser API calls are handled by the Blazor runtime via JavaScript interop.
+
+To support older browsers that don't support WebAssembly, use the [server-side hosting model](#server-side-hosting-model).
+
+For more information, see <xref:razor-components/hosting-models#client-side-hosting-model>.
+
 ## Infrastructure
 
-Razor Components offers the core facilities that most apps require, including:
+Core facilities that most apps require are provided by the framework, including:
 
 * Layouts
 * Routing
 * Dependency injection
 
-All of these features are optional. When one of these features isn't used in an app, the implementation is stripped out of the app when published by the [Intermediate Language (IL) Linker](xref:host-and-deploy/razor-components/configure-linker).
+These scenarios are optional. When one of these scenarios isn't used by a Blazor app, the implementation is stripped out of the app when it's published by the [Intermediate Language (IL) Linker](xref:host-and-deploy/razor-components/configure-linker).
 
 ## Code sharing and .NET Standard
 
 Apps can reference and use existing [.NET Standard](/dotnet/standard/net-standard) libraries. .NET Standard is a formal specification of .NET APIs that are common across .NET implementations. .NET Standard 2.0 or higher is supported. APIs that aren't applicable inside a web browser (for example, accessing the file system, opening a socket, threading, and other features) throw [PlatformNotSupportedException](/dotnet/api/system.platformnotsupportedexception). .NET Standard class libraries can be shared across server code and in browser-based apps.
 
 ## JavaScript interop
 
-For apps that require third-party JavaScript libraries and browser APIs, WebAssembly is designed to interoperate with JavaScript. Razor Components are capable of using any library or API that JavaScript is able to use. C# code can call into JavaScript code, and JavaScript code can call into C# code. For more information, see [JavaScript interop](xref:razor-components/javascript-interop).
+For apps that require third-party JavaScript libraries and browser APIs, Razor Components interoperate with JavaScript. Razor Components are capable of using any library or API that JavaScript is able to use. C# code can call into JavaScript code, and JavaScript code can call into C# code. For more information, see [JavaScript interop](xref:razor-components/javascript-interop).
 
 ## Optimization
```

Comment:
Again, this section is specific to Blazor and should probably move to a Blazor specific topic.